singleheader/amalgamate.sh

```diff
@@ -108,7 +108,7 @@ mkdir -p $AMALGAMATE_OUTPUT_PATH
 AMAL_H="${AMALGAMATE_OUTPUT_PATH}/simdjson.h"
 AMAL_C="${AMALGAMATE_OUTPUT_PATH}/simdjson.cpp"
 DEMOCPP="${AMALGAMATE_OUTPUT_PATH}/amalgamate_demo.cpp"
```

Comment:
Does this still need to be here? I can't find where it's used.

Reply:
In my first change I removed it, but it appears that all Ci tests run things from some other dir and readme and democpp are accessed from that other tmp build dir. To keep readme the same (so that it wouldn't need to be changed in CI or when running in the same dir) and to keep the same democpp I copy it using `cp -u` to copy only if the destination is older

Reply:
Ah yes. The right way to do this is to add those copies to the CMakeLists.txt instead of the amalgamator, but we won't block this checkin just for that :)

Reply:
The build directory is where we get the files from when we make packages. I think the README still needs to be copied there. Please add the copy and ls -al in here, if you could. Also set README="${AMALGAMATE_OUTPUT_PATH}/README.md" instead of SCRIPTPATH for consistency.

Then I think we're good and done, thanks for this!